plugins/org.yakindu.sct.ui.editor/src/org/yakindu/sct/ui/editor/validation/SCTValidationJob.java

```diff
@@ -120,45 +111,23 @@ public boolean isCanceled() {
 			try {
 				editingDomain.runExclusive(runner);

```

Comment:
Please make this overrideable by extracting to a protected method.
In my case, I have to run the validation e.g. in a write transaction and rollback afterwards.